### DISCUSSION TITLE
Sync Agenda with Google Calendar and Microsoft Agenda

### DISCUSSION THREAD

**Discussion by vitorpadilha** [exoplatform/exo-community#58] [Q&A]
Hello,

   I configured the eXo Platform to use the  Calendar Service with the user log in with a Google or Microsoft Account with oAuth. Perhaps that only tasks joined in the platform was loaded in Google Calendar (or Microsoft). If i put the task directly in the Google Calendar website (or App), the task don't appear in the eXo Platform. Therefore, the sync is in one-way (eXo to Google Calendar). Are there a configuration that my tasks registered in Google Calendar appear in the Platform?

Thanks.

**Comment by rdenarie** [comment]
Hello @vitorpadilha 
I config that the synchronization is only from eXo to Google.

The only way you can see google events in eXo is on the details of an event : on the right part, there is the list of the event coming from external calendars which are displayed
